solutions/secure/README.md

```diff
@@ -0,0 +1,8 @@
+# IBM Elastic Search on IBM Cloud
+
+This architecture creates an IBM Elastic Search instance in IBM Cloud environment and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Elastic search instance on IBM Cloud
+
```

Comment:
Could you add the following note:

**NB:** This solution is not intended to be called by one or more other modules since it contains a provider configurations, meaning it is not compatible with the `for_each`, `count`, and `depends_on` arguments. For more information see [Providers Within Modules](https://developer.hashicorp.com/terraform/language/modules/develop/providers)